### DISCUSSION TITLE
TaskType for Named Entity Recognition

### DISCUSSION THREAD

**Discussion by AndreaLombax** [huggingface/peft#861] [General]
Hi,
I'm trying to train a LLM to perform NER on some text lines. Without going into further details, a doubt came to me: which task_type should I set for LoraConfig?
These are the possible tasks:
`SEQ_CLS, SEQ_2_SEQ_LM, CAUSAL_LM, TOKEN_CLS, QUESTION_ANS, FEATURE_EXTRACTION.`

**Comment by BenjaminBossan** [comment]
I think `TOKEN_CLS` should be used for NER: https://huggingface.co/docs/transformers/tasks/token_classification
